assets/consts.js

```diff
@@ -1,3 +1,9 @@
+export const DEFAULT_LANGUAGE = 'en';
+export const URL_PREFIX = '/b';
+export const DEFAULT_TRANSLATION = 'pl_pubg';
```

Comment:
Since English is the default language now then maybe we should replace this with `en_kjv`?

Also I was thinking that maybe we could use some default settings depending on the selected language? wdyt?

Reply:
This is actually the only question which is holding me before merging your PR.

Reply:
> Since English is the default language now then maybe we should replace this with en_kjv?
It works slightly differently. The language is taken from the browser setting. If there is no language specified in the browser than the English is selected. At least this behaviour I wanted to implement.

> Also I was thinking that maybe we could use some default settings depending on the selected language? wdyt?

Sounds like an interesting feature to implement in the future.

Reply:
Cool, thanks for clarification :)